storage/acceptance/adls_gen2_test.go

```diff
@@ -45,42 +44,3 @@ func TestAzureAccAdlsGen2Mount_correctly_mounts(t *testing.T) {
 		},
 	})
 }
-
-func TestAzureAccAdlsGen2Mount_capture_error(t *testing.T) {
-	if _, ok := os.LookupEnv("CLOUD_ENV"); !ok {
-		t.Skip("Acceptance tests skipped unless env 'CLOUD_ENV' is set")
-	}
-	client := common.CommonEnvironmentClient()
-	if !client.AzureAuth.IsClientSecretSet() {
-		t.Skip("Test is meant only for SP Azure")
-	}
-	acceptance.AccTest(t, resource.TestCase{
```

Comment:
hey can you just reply back to this comment in regards to why we removed this test?

Reply:
Removed TestAzureAccAdlsGen2Mount_capture_error because it's covered by TestAccInvalidSecretScopeFails already

https://github.com/databrickslabs/terraform-provider-databricks/pull/314/commits/c24700289e712a8ed456ee908c22d95b598c4a1a

Reply:
And the tests were failing with tf13 backend